docs/mkdocs.yml

```
@@ -108,4 +110,8 @@ markdown_extensions:
           format: !!python/name:pymdownx.superfences.fence_code_format
 plugins:
   - glightbox
-  - git-authors
\ No newline at end of file
+  - git-authors
+  - swagger-ui-tag:
+        docExpansion: none
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 8

(indentation)